relayer/src/services/relayService/addRelayJob.ts

```diff
@@ -0,0 +1,16 @@
+import { TransactionInstruction } from "@solana/web3.js";
+import { relayQueue } from "../../db/redis";
+
+export async function addRelayJob({
```

Comment:
I deleted this file on purpose, to consolidate files and remove directories if we can,
Personally I don't like to have a lot of files with  less than 20 lines each.